quick.html

```diff
@@ -52,8 +53,13 @@
 <i>throw away code</i>. We're sure that once you get used to the
 WebDriver API, you will never use Selenium IDE.
 
-
-<h2>Selenium Grid</h2>
+<p>The Selenium project team <a href="https://seleniumhq.wordpress.com/2017/08/09/firefox-55-and-selenium-ide/">announced</a>
+ in August 2017 that Selenium IDE is 
+not compatible with Firefox 55 and future versions of the web browser.
+Hence, Selenium IDE will no longer work. The two main causes were : 
+ Firstly, Firefox is switching extensions from the "XPI" format to more widely 
+ <a href="https://blog.mozilla.org/addons/2016/11/23/add-ons-in-2017/">"Web Extension"</a> mechanism. Secondly, The project lacks time and energy to move 
+ the IDE forward and take advantage of the new technologies.
```

Comment:
We should mention that there is a new IDE in development. There's no link right now, but the repo is at `https://github.com/seleniumhq/selenium-ide`

Reply:
@shs96c I hope it gets released sooner :)